app/views/provider/admin/account/payment_gateways/braintree_blue/edit.html.slim

```diff
@@ -1,3 +1,40 @@
+css:
+  .form-control:- moz -placeholder {
+    color: #999999;
+      }
+      .form-control::-moz-placeholder {
+        color: #999999;
+          }
+          .form-control:- ms - input -placeholder {
+            color: #999999;
+              }
+              .form-control::-webkit-input-placeholder {
+                color: #999999;
+                  }
+                  .form-control {
+                    display: block;
+                      width: 100%;
```

Comment:
This indentation kills me ...

Reply:
Indeed 😂 Auto indented in: https://github.com/3scale/porta/pull/896